propertysuggester/parser/JsonReader.py

```diff
@@ -61,21 +61,21 @@ def _process_json(data):
     if "claims" not in data:
         return Entity(title, [])
     claims = []
-    for property_id, statements in data["claims"].items():
+    for property_id, statements in sorted(list(data["claims"].items())):
```

Comment:
A random question: does it matter that those statements will now be parsed in the property ID order not in the order they were in the input data? Or is the json loader getting this data in non-deterministic order any way?

Reply:
Order doesn't matter but I want to keep some order (It's not important what type of order) to make it consistent so tests don't fail randomly. Because dict.items() in python returns things in random order.

Reply:
Clear. Thanks!